toolkit/tools/imagecustomizerapi/systemconfig.go

```diff
@@ -56,5 +57,13 @@ func (s *SystemConfig) IsValid() error {
 		}
 	}
 
+	allServices := append(s.Services.Enable, s.Services.Disable...)
+	for i, service := range allServices {
+		err = service.IsValid()
+		if err != nil {
+			return fmt.Errorf("invalid Service item at index %d: %w", i, err)
```

Comment:
You should have separate `for` loops for `Enable` and `Disable`. This will ensure that the index (`i`) is correct for the `Disable` list.

Also, it would be good to add which list the item is in within the error message. (e.g. `Service.Enable` and `Service.Disable`).